Languages/french.lang

```diff
@@ -72,25 +72,25 @@ msgid "1 hour"
 msgstr "1 heure"
 
 msgid "10 min"
-msgstr "10 min."
+msgstr "10 min"
 
 msgid "2 (Teen 12+)"
 msgstr "2 (Adolescents 12+)"
 
 msgid "20 min"
-msgstr "20 min."
+msgstr ""
 
 msgid "2D Cover Path"
 msgstr "Dossier jaquettes 2D"
 
 msgid "3 (Mature 16+)"
-msgstr "3 (Matures 16+)"
+msgstr "3 (Jeunes adultes 16+)"
 
 msgid "3 min"
-msgstr "3 min."
+msgstr ""
```

Comment:
```suggestion
msgstr "3 min"
```

Reply:
I think if empty this is default native English language that is used but in doubt ...